test/sslprovidertest.c

```diff
@@ -114,15 +136,14 @@ int setup_tests(void)
         return 0;
     }
 
+    defctxlegacy = OSSL_PROVIDER_load(NULL, "legacy");
```

Comment:
Should the block comment move down here along with the `OSSL_PROVIDER_load(NULL, .)`?  It seems to still be most relevant here.

Reply:
Done